dev-db/influxdb/influxdb-2.7.11.ebuild

```diff
@@ -194,11 +194,12 @@ inherit cargo go-module systemd
 DESCRIPTION="Scalable datastore for metrics, events, and real-time analytics"
 HOMEPAGE="https://www.influxdata.com"
 
-FLUX_PV="0.195.1"
+FLUX_PV="0.195.2"
 
 SRC_URI="https://github.com/influxdata/influxdb/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-SRC_URI+=" https://github.com/influxdata/ui/releases/download/OSS-2.7.8/build.tar.gz -> ${P}-assets.tar.gz"
+SRC_URI+=" https://github.com/influxdata/ui/releases/download/OSS-v2.7.10/build.tar.gz -> ${P}-assets.tar.gz"
```

Comment:
(Why .10, not .11?)

Reply:
Intentional, ui version and the actual package version are not necessarily in sync influxdata/influxdb#25504

Reply:
Let's have it as a comment then please.

Reply:
Exactly, the version of the UI is not necessarily the same.

Reply:
> Let's have it as a comment then please.

I was thinking in creating a UI_PV variable to make it more explicit it's a different component instead of adding a new comment. It's fine?

Reply:
wfm